po/pl.po

```diff
@@ -222,33 +221,31 @@ msgid "`%s' is not a valid configuration element"
 msgstr "`%s' nie jest poprawnym elementem konfiguracji"
 
 #: src/configcontainer.cpp:159
-#, fuzzy, c-format
+#, c-format
 msgid "Newsboat: finished reload, %f unread feeds (%n unread articles total)"
 msgstr ""
-"newsboat: odświeżanie zakończone, %f nieprzeczytanych kanałów(%n "
+"Newsboat: odświeżanie zakończone, %f nieprzeczytanych kanałów(%n "
```

Comment:
Just noticed there's no space before the opening parenthesis, is that intentional? ;)

Reply:
Nope. Nice catch! :)